wolfcrypt/src/chacha.c

```diff
@@ -408,12 +420,10 @@ static void chacha_encrypt_avx(ChaCha* ctx, const byte* m, byte* c,
     byte*  output;
     word32 i;
     word32 cnt = 0;
-    static const word64 add[2] =  { 0x0000000100000000UL,0x0000000300000002UL };
-    static const word64 four[2] = { 0x0000000400000004UL,0x0000000400000004UL };
-    static const word64 rotl8[2] =
-                                  { 0x0605040702010003UL,0x0e0d0c0f0a09080bUL };
-    static const word64 rotl16[2] =
-                                  { 0x0504070601000302UL,0x0d0c0f0e09080b0aUL };
```

Comment:
Hey Sean, in this PR did you mean to go back to using `__m128i`? A previous PR #1081 changed this to use the word64. https://github.com/wolfSSL/wolfssl/pull/1081